trin-history/Cargo.toml

```diff
@@ -27,3 +27,8 @@ tracing = "0.1.26"
 tracing-subscriber = "0.2.18"
 tokio = { version = "1.8.0", features = ["full"] }
 trin-core = { path = "../trin-core" }
+
+[dev-dependencies]
+env_logger = "0.8.2"
```

Comment:
For `trin-core` we only import `test-log` and `tracing-subscriber`. Based on the [`test-log` crate docs](https://crates.io/crates/test-log), we only need one of `test-log` or `env_logger`, so I'm not sure why we add both as dependencies here.

Reply:
I see. I figured that there was some reason for the inclusion of `env_logger`.